### ISSUE TITLE
Logger outputs duplicate in console logging: 'Logger.info("")'

### ISSUE THREAD

**Issue by Joepeman** [cookingfox/logging-adapter-java#1]
``` java
Logger.info("");
Logger.info("ABC");
```

Expected:

```
09-17 14:32:25.400  28812-28812/com.cookingfox.cfeventdemo I/ManagerBackgroundService#onCreate:31﹕ 
09-17 14:32:25.400  28812-28812/com.cookingfox.cfeventdemo I/ManagerBackgroundService#onCreate:32﹕ ABC
```

Actual:

```
09-17 14:33:54.537  30213-30213/com.cookingfox.cfeventdemo I/ManagerBackgroundService#onCreate:31﹕ [ 09-17 14:33:54.537 30213:30213 I/ManagerBackgroundService#onCreate:32 ]
    ABC
```

**Comment by abelcookingfox** [comment]
It looks like this behaviour is in Android's `logcat` formatting - if you try Android's native `Log` class, the same thing happens when an empty string is pasted: [StackOverflow](http://stackoverflow.com/questions/32143903/android-logcat-very-strange-behavior-when-getting-empty-string).

Quote:

> Based on my quick read through things what I think is happening is the message is not terminated properly. The empty message does not show up until another message is appended and the first message overruns and shows the second message because it has the appropriate termination.

This issue poses another question: what to print when the log message is empty? I would say just print nothing, which works fine for the other logger adapters in a non-Android environment. So, I propose to print something like `[empty log message]` when using the `AndroidLoggerAdapter`.

@Joepeman What do you think?

**Comment by abelcookingfox** [comment]
This is fixed in version 0.3.1: a9a7af0dc3d13a7c85d84114ef0a2093dced8816
